migrations/sql/V2018.11.20.1106__AddModelMeta.sql

```diff
@@ -0,0 +1,60 @@
+--- What this Migration is about
+--- comment on model.is_current
+--- &
+--- creating three new table
+--- 1. model_version_meta
+--- 2. model_version_vaccine
+--- 3. model_version_country
+
+COMMENT ON COLUMN model.is_current IS
+  'TRUE if the model is currently running for VIMC.';
+
+CREATE TABLE model_version_meta (
+  id SERIAL,
+  model_version INTEGER NOT NULL,
+  is_dynamic BOOLEAN NOT NULL, 
+  code TEXT,
+  dalys BOOLEAN NOT NULL,
+  gender INTEGER NOT NULL,
+  burden_min_age INTEGER NOT NULL,
+  burden_max_age INTEGER NOT NULL,
+  cohort_first INTEGER, --- nullable, NA means 2000
```

Comment:
Why nullable meaning 2000? why not set to 2000 in the import? If that's for some reason not possible or desirable please explain and then document the NA meaning in the comment for the field (and the same for all other columns with "meaningful" NA values)

Reply:
I am making them explicit. Done.